src/main.rs

```diff
@@ -786,9 +790,12 @@
 //! Available for the _init_ (`cargo wix init`) and _print_ (`cargo wix print`)
 //! subcommands.
 //!
-//! Sets the path to a bitmap (.bmp) image file that will be displayed across
+//! Sets the path to an image file (.png, .bmp, ...) that will be displayed across
```

Comment:
Are we sure a PNG can be used? Maybe PNG support was added to WiX after I originally implemented this feature, but I vaguely remember only bitmaps being supported.

Reply:
Yeah even WiX's docs don't say it's allowed but I tested it out and png's work great (and should probably be preferred for obvious reasons).